app/templates/components/main-map.hbs

```diff
@@ -98,12 +91,5 @@
   {{mapbox-gl-on 'data' (action 'mapLoading') eventSource=map.instance}}
 {{/mapbox/basic-map}}
 
-{{!--
-  TODO: Geolocate Feature -> Move into separate component
---}}
-{{#unless findMeDismissed}}
-  <div class="find-me">
-    <button {{action 'locateMe'}} class="button hide-for-medium hide-for-print no-margin">Locate&nbsp;Me</button>
-    <button {{action 'dismissFindMe'}} class="button hide-for-medium hide-for-print no-margin">&times;</button>
-  </div>
-{{/unless}}
+{{#locate-me-mobile}}
```

Comment:
why is this being invoked as a block component if nothing is invoked/rendered inside it?